Mathlib/Probability/Distributions/Uniform.lean

```diff
@@ -28,35 +35,88 @@ This file defines a number of uniform `PMF` distributions from various inputs,
 
 `PMF.ofMultiset` draws randomly from the given `Multiset`, treating duplicate values as distinct.
   Each probability is given by the count of the element divided by the size of the `Multiset`
+
+# To Do:
+* Refactor the `PMF` definitions to come from a `uniformMeasure` on a `Finset`/`Fintype`/`Multiset`.
 -/
 
 open scoped Classical MeasureTheory BigOperators NNReal ENNReal
 
-open TopologicalSpace MeasureTheory.Measure
+open TopologicalSpace MeasureTheory.Measure PMF
+
+noncomputable section
 
 namespace MeasureTheory
 
 variable {E : Type*} [MeasurableSpace E] {m : Measure E} {μ : Measure E}
-variable {Ω : Type*} {_ : MeasurableSpace Ω} {ℙ : Measure Ω}
 
+/-- A measure is a uniform measure for a set `s` if it is the rescaled restriction of the measure to
+this set.  -/
+def uniformMeasure (s : Set E) (μ : Measure E := by volume_tac) : Measure E :=
```

Comment:
Actually, I think this is the same as [`ProbabilityTheory.cond`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Probability/ConditionalProbability.html#ProbabilityTheory.cond). Can you use it instead?

Reply:
Good catch, let me change that indeed!